presto-spi/src/main/java/com/facebook/presto/spi/plan/TableScanNode.java

```diff
@@ -68,14 +71,16 @@ public TableScanNode(
             TableHandle table,
             List<VariableReferenceExpression> outputVariables,
             Map<VariableReferenceExpression, ColumnHandle> assignments,
-            TupleDomain<ColumnHandle> currentConstraint,
-            TupleDomain<ColumnHandle> enforcedConstraint)
+            boolean isDelete,
+            TupleDomain<ColumnHandle> enforcedConstraint,
```

Comment:
Why do we flip the `enforcedConstraint` with the `currentConstraint` in here?

Reply:
unintentional.  i'll flip it back.